### ISSUE TITLE
Accent folding research/how-to

### ISSUE THREAD

**Issue by underlog** [telerik/kendo-ui-core#1651]
Technically, this is not in our scope, but is a common problem when it comes to data filtering - I think that we can wrap up a help article that points the people to some recommended approaches. 

Request: http://kendoui-feedback.telerik.com/forums/127393-telerik-kendo-ui-feedback/suggestions/5393746-accent-folding-for-kendo-combo-box

**Comment by sylvain-hamel** [comment]
Would you accept a PR that adds support for case folding?

**Comment by underlog** [comment]
Yes, but to make sure that no effort gets wasted, it would be great if we can discuss the implementation approach first. Do you have anything in mind? Some sort of reference implementation would be the best.

**Comment by sylvain-hamel** [comment]
Hi, I'd add an `ignoreAccents` option to the dropdown that people could activate the same way `ignoreCase` can be activated. The code to fold actually the accent would be where `ignoreCase` is evaluated in `Query.filterExpr`.

**Comment by ggkrustev** [comment]
Hi @sylvain-hamel,

After a detailed research, we decided to put off the built-in support for "accent folding", at least for now. The reason for this decision is the fact that adding an `ignoreAccent` field would not solve the issue, because the filtering operators, e.g. equal, greater-than and etc., don't know how to treat it. The specific implementation would require to maintain a list of accent tables, which is not an easy task at all and an error prone too.

The best solution, would be to describe how the filtering could be done and the developer that would need to support specific accent to provide the concrete implementation. That being said, we documented the possible ways to handle the data source filtering and where and how to apply the accent folding. Find more details in the corresponding help article:

- https://github.com/telerik/kendo-ui-core/blob/master/docs/framework/datasource/basic-usage.md#accent-folding

(the documentation is still not finalized)

Relevant commit that added the "Accent folding" details: 51edbac738dc0113583833611e0e4a6578bd7184

I will close this thread with that message.
